lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -219,7 +219,7 @@ def self.get_release(api_token, release_url)
       end
 
       # add release to distribution group
-      def self.add_to_group(api_token, release_url, group_name, release_notes = '')
+      def self.add_to_group(api_token, release_url, group_name, release_notes = '', notify_testers = true)
```

Comment:
I would personally vote for the default here being `false` since that makes more sense to me in a CI scenario like where the fastlane plugin is likely used.
However, I believe this would be a breaking change to the current default behavior without specifying the flag explicitly which does always trigger the notification.

Reply:
I think so too, currently it always triggers the notification. Which is why I did not want to change the behaviour. But I also see your point in the CI case. How would do you guys handle this? I'm open for both.